openforcefield/utils/toolkits.py

```diff
@@ -2127,84 +2078,33 @@ def to_rdkit(molecule, aromaticity_model=DEFAULT_AROMATICITY_MODEL):
-                    if n != bond.atom1
-                ][0]
-                # Get rdmol bonds
-                bond_atom1_index = molecule.atoms.index(bond.atom1)
-                bond_atom2_index = molecule.atoms.index(bond.atom2)
-                bond1 = rdmol.GetBondBetweenAtoms(map_atoms[n1],
-                                                  map_atoms[bond.atom1_index])
-                bond2 = rdmol.GetBondBetweenAtoms(map_atoms[bond_atom1_index],
-                                                  map_atoms[bond.atom2_index])
-                bond3 = rdmol.GetBondBetweenAtoms(map_atoms[bond_atom2_index],
-                                                  map_atoms[n2])
-                # Set arbitrary stereochemistry
-                # Since this is relative, the first bond always goes up
-                # as explained above these names come from SMILES slashes so UP/UP is Trans and Up/Down is cis
-                bond1.SetBondDir(Chem.BondDir.ENDUPRIGHT)
-                bond3.SetBondDir(Chem.BondDir.ENDDOWNRIGHT)
-                # Flip the stereochemistry if it is incorrect
-                # TODO: Clean up _CIPCode atom and bond properties
-                Chem.AssignStereochemistry(rdmol, cleanIt=True, force=True)
-                if bond.stereochemistry == 'E':
-                    desired_rdk_stereo_code = Chem.rdchem.BondStereo.STEREOE
-                elif bond.stereochemistry == 'Z':
-                    desired_rdk_stereo_code = Chem.rdchem.BondStereo.STEREOZ
-                else:
-                    raise Exception(
-                        "Unknown bond stereochemistry encountered in "
-                        "to_rdkit : {}".format(bond.stereochemistry))
-
-                if bond2.GetStereo() != desired_rdk_stereo_code:
-                    # Flip it
-                    bond3.SetBondDir(Chem.BondDir.ENDUPRIGHT)
-                    # Validate we have the right stereochemistry as a sanity check
-                    Chem.AssignStereochemistry(rdmol, cleanIt=True, force=True)
-                    #if rdmol.GetProp('_CIPCode') != bond.stereochemistry:
-                    if bond2.GetStereo() != desired_rdk_stereo_code:
-                        raise Exception(
-                            'Programming error with assumptions about RDKit stereochemistry model'
-                        )
+        # Copy bond stereo info from molecule to rdmol.
+        cls._assign_rdmol_bonds_stereo(molecule, rdmol)
 
         # Set coordinates if we have them
         if molecule._conformers:
             for conformer in molecule._conformers:
                 rdmol_conformer = Chem.Conformer()
-                for index, rd_idx in map_atoms.items():
-                    (x, y, z) = conformer[index, :] / unit.angstrom
-                    rdmol_conformer.SetAtomPosition(rd_idx,
+                for atom_idx in range(molecule.n_atoms):
+                    (x, y, z) = conformer[atom_idx, :] / unit.angstrom
```

Comment:
With what I now know about simtk units, I think it's safer to make this `.in_units_of(unit.angstrom)`

Reply:
Good idea. Will change!